thrift/src/main/thrift/atoms/interactive.thrift

```diff
@@ -0,0 +1,13 @@
+namespace * contentatom.cta
+namespace java com.gu.contentatom.thrift.atom.interactive
+#@namespace scala com.gu.contentatom.thrift.atom.interactive
+
+struct InteractiveAtom {
+  /* the unique ID will be stored in the `atom` data*/
+  2: required string type
```

Comment:
So for explainer this stores the type of explainer, there can either be an expandable explainers or just plain explainers.

It's up to the individual atom to decide if there are different types of s specific atom, but it's definitely not required.